src/action.c

```diff
@@ -674,7 +674,8 @@ actions_run(struct view *activator, struct server *server,
 			if (view) {
 				/* Config parsing makes sure that direction is a valid direction */
 				enum view_edge edge = action_get_int(action, "direction", 0);
-				view_snap_to_edge(view, edge, /*store_natural_geometry*/ true);
+				view_snap_to_edge(view, edge, true,
+					/*store_natural_geometry*/ true);
```

Comment:
Could you add a inline comment for the new bool here? It makes it way easier to read the code rather than having a random assortment of `true` / `false` arguments.

Same for the call in `interactive.c`.